server/modules/record/manager/_recordManager/recordUniquenessValidator.js

```diff
@@ -49,19 +47,19 @@ const validateEntityKeysUniqueness = (survey, record, nodeEntity) => {
   )(entityValidations)
 }
 
-const isDuplicateEntity = (survey, record, siblingEntitiesAndSelf, entity) => {
+const _isEntityDuplicated = (survey, record, siblingEntitiesAndSelf, entity) => {
```

Comment:
it should be "_isEntityDuplicate" not "duplicateD"...

Reply:
not sure: http://conjugator.reverso.net/conjugation-english-verb-duplicate.html

Reply:
duplicate is an [adjective](https://www.dictionary.com/browse/duplicated)!

Reply:
changed!